package.json

```diff
@@ -1,6 +1,6 @@
 {
-  "name": "apollo-link-rest",
```

Comment:
Obviously we're not going to be able to merge any of these changes to package.json :P

Reply:
Lol, that's was unintentional :p